app/frontend/app/components/group-box.js

```diff
@@ -0,0 +1,51 @@
+import Ember from 'ember';
+/* global Messenger */
+
+export default Ember.Component.extend({
+
+  userIsMemberOfGroup: function(){
```

Comment:
This is something that we should be serializiing from the server, not computing client side. See how we handle returning whether the current user is following someone for an example.